### ISSUE TITLE
List applications?

### ISSUE THREAD

**Issue by austinmw** [halcy/Mastodon.py#375]
Hi, after using `Mastodon.create_app`, is there a way to list applications that have been created?

**Comment by codl** [comment]
No, there is no such API in Mastodon. You must save the credentials returned by `create_app` somewhere so you can reuse them later.
